frontend/src/individual_certificate.tsx

```diff
@@ -244,10 +251,55 @@ export function IndividualCertificate({ id }: { id: string }) {
           );
         } else if (res.status === 404) {
           throw new Error("Kaninen kunde inte hittas.");
-        } else res.blob();
+        } else if (res.status === 200) {
+          return res.blob();
+        } else {
+          throw new Error("Något gick fel.");
+        }
+      })
+      .then((blob) => {
+        if (blob) {
+          setCertificateUrl(window.URL.createObjectURL(blob));
+          setShowSummary(false);
+          setShowComplete(true);
+        } else {
+          throw new Error("Något gick fel.");
```

Comment:
```suggestion
          throw new Error("Något gick fel (det här borde inte hända).");
```

?

This is one problematic area - for translation work, it's helpful to use the same string when possible. At the same time, having differing messages is useful for getting more helpful bug reports at times.